_includes/package-grid.html

```diff
@@ -52,6 +52,9 @@ <h3 class="card__title no_toc" itemprop="headline">
                 <!-- TODO: Create and use sunpy.html here -->
                 <li><a href="communities/astropy.html"><i class="fa-solid fa-check-double"></i> <img src="http://img.shields.io/badge/Affiliated-SunPy-yellow.svg?style=flat" alt="SunPy" /></a></li>
             {% endif %}
+            {% if apackage.active == false %}
```

Comment:
@banesullivan this is a good start. 

https://output.circle-artifacts.com/output/job/9350867b-9505-4a09-b082-a278c9751dd3/artifacts/0/_site/python-packages.html

What we should do here since we only have 1 archived packages is create a new section at the bottom of that page that says .
```
## Archived packages 
Archived packages are packages that have successfully completed [pyOpenSci's software peer review process](https://www.pyopensci.org/about-peer-review/index.html) but are no longer maintained. 
```

then create a for loop using jekyll / liquid that grabs those packages. Rather than a card, let's list them with the package name and a link to the review for now. 

that way, we aren't modifying the package-grid template. We are just adding a new section and listing archived pages. in the future, they may live on a separate page, but for now, let's list them at the bottom without a card.